hypervisor/src/mshv/mod.rs

```diff
@@ -13,6 +13,18 @@
 #[cfg(target_arch = "x86_64")]
 pub mod x86_64;
 
+use mshv_bindings::Msrs as MsrEntries;
+pub use mshv_bindings::*;
+use mshv_ioctls::{set_registers_64, Mshv, VcpuFd, VmFd};
+
+/// Vcpu struct for Hyper-V
```

Comment:
Hyper-V should be Microsoft Hypervisor. Please also fix other occurrences in code and commit message.

Reply:
And again, there is an extraneous line between two SoB lines.